app/src/androidTest/java/org/mozilla/fenix/helpers/Matchers.kt

```diff
@@ -56,3 +60,13 @@ fun nthChildOf(
         }
     }
 }
+
+fun ViewInteraction.isVisibleForUser(): Boolean {
+    try {
+        check(matches(ViewMatchers.isCompletelyDisplayed()))
```

Comment:
@kglazko / @rpappalax are you aware of any issues with using this over something like `scrollToElementByText`? 🤔

Reply:
The problem that I encountered here was that on all test methods from `SettingsSubMenuAboutRobot` scroll was performed (example: TestHelper.scrollToElementByText("What’s new in Firefox Preview")). This test class was flaky because on larger devices it could not perform this scroll and it would throw an error (this approach worked only for smaller devices).

That being said I had to find a solution to scroll only when needed. Another problem was that `perform(click())` will work only if the view is at least 90% visible, so I had to make sure that the view is completely visible. This is why I used `isVisibleForUser` extension where I check if `ViewMatchers.isCompletelyDisplayed()`.